uaclient/entitlements/repo.py

```diff
@@ -204,17 +218,20 @@ def process_contract_deltas(
             "Updating '%s' apt sources list on changed directives.", self.name
```

Comment:
As you mentioned in chat, this message shouldn't be emitted if we actually aren't rewriting apt source config files. Let's move this into scope below ```        if not self._check_apt_url_is_applied(delta_apt_url):```